src/charm.py

```diff
@@ -778,12 +832,37 @@ def _run_wp_cli(self, cmd, timeout=60, combine_stderr=False):
             group=self._WORDPRESS_GROUP,
             working_dir="/var/www/html",
             combine_stderr=combine_stderr,
-            timeout=timeout)
+            timeout=timeout
+        )
         try:
             stdout, stderr = process.wait_output()
-            return Result(0, stdout, stderr)
+            result = Result(0, stdout, stderr)
         except ops.pebble.ExecError as e:
-            return Result(e.exit_code, e.stdout, e.stderr)
+            result = Result(e.exit_code, e.stdout, e.stderr)
+        if combine_stderr:
+            logger.debug(f"Run wp-cli command: {cmd}\noutput: {result.stdout}")
+        else:
+            logger.debug(
+                f"Run wp-cli command: {cmd}\nstdout: {result.stdout}\nstderr:{result.stderr}"
+            )
+        return result
+
+    def _wrapped_run_wp_cli(self, cmd, timeout=60, error_message=None):
+        """Run wp cli command and return the result as ``self._ExecResult``
```

Comment:
are the double quotes ``  around self._ExecResult intended here?

Reply:
This is reStructuredText syntax https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
Ah I see